src/v/kafka/server/handlers/describe_configs.cc

```diff
@@ -115,11 +116,17 @@ static void add_broker_config(
         }
     }
 
+    ss::sstring doc = include_documentation ? "doc string" : "";
+
+    describe_configs_type config_type = describe_configs_type::unknown;
```

Comment:
I don't understand where the `documentation` value will come from
nor do I understand how the `config type` value is determined

Reply:
The documentation and config type are dependent on the resource.
So the documentation should use the `desc` from the config store for that particular property.
And the config type is based on the properties type as well.